EU4toV2/Source/V2World/V2Reforms.cpp

```diff
@@ -33,20 +33,17 @@ SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.*/
 
 V2Reforms::V2Reforms(const V2Country* dstCountry, const std::shared_ptr<EU4::Country> srcCountry)
 {
-	slavery							= 0;
-	vote_franchise					= 0;
-	upper_house_composition		= 0;
-	voting_system					= 0;
-	public_meetings				= 0;
-	press_rights					= 0;
-	trade_unions					= 0;
-	political_parties				= 0;
-	upperHouseEffects(dstCountry);
-	governmentEffects(dstCountry);
-
 	if ((srcCountry->hasModifier("the_abolish_slavery_act")) || (srcCountry->hasModifier("abolish_slavery_act")))
 	{
-		slavery += 1;
+		abolishSlavery = true;
+		slavery = dstCountry->getSlavery();
+		upper_house_composition = dstCountry->getUpper_house_composition();

```

Comment:
Why would most of these be only in the conditional?

Reply:
jesus, they would not. Fixed.